test/Runtime/environment_variables.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-build-swift %s -o %t/main
+// RUN: %target-codesign %t/main
+// RUN: env %env-SWIFT_HELP=YES env %env-SWIFT_SOME_UNKNOWN_VARIABLE=42 %env-SWIFT_DEBUG_ENABLE_METADATA_ALLOCATION_ITERATION=YES %env-SWIFT_DEBUG_IMPLICIT_OBJC_ENTRYPOINT=abc %env-SWIFT_DETERMINISTIC_HASHING=whatever %env-SWIFT_ENABLE_MANGLED_NAME_VERIFICATION=YES %target-run %t/main 2>&1 | %FileCheck %s --dump-input fail
```

Comment:
There is an extra `env` in between `%env-SWIFT_HELP=YES` and `%env-SWIFT_SOME_UNKNOWN_VARIABLE=42`.